types/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for Chalk 2
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export = chalk;
+
+declare function chalk(...text: (string | number)[]): string;
+declare function chalk(text: TemplateStringsArray, ...placeholders: (string | number)[]): string;
+
+declare namespace chalk {
+
+	interface ChalkConstructorOptions {
+		enabled?: boolean;
+		level?: number;
+	}
+
+	export function constructor(options?: ChalkConstructorOptions): typeof chalk;
+
+	export let enabled: boolean;
+	export let level: number;
+	export const supportsColor: boolean;
```

Comment:
Is it possible to define this as just an re-export of the [`supports-color`](https://github.com/chalk/supports-color) package? I guess this could be useful if users wants to Cmd+click to view source definition or something.